ERCS/erc-9988.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 9988
```

Comment:
```suggestion
eip: 7661
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

Reply:
I've already bought the domain name & hosted the website though.

Reply:
I've already been posting about it aswell. I don't get why I have to change it when there are randomly chosen EIP numbers.